qanary-component-MT-Python-MBart/requirements.txt

```diff
@@ -0,0 +1,93 @@
+alembic
+attrs
+bcrypt
+certifi
+cffi
+charset-normalizer
+click
+cloudpickle
+contourpy
+cryptography
+cycler
+databricks-cli
+docker
+entrypoints
+exceptiongroup
+filelock
+Flask
+fonttools
+gitdb
+GitPython
+greenlet
+gunicorn
+huggingface-hub
+idna
+importlib-metadata
+iniconfig
+isodate
+itsdangerous
+Jinja2
+joblib
+kiwisolver
+langdetect
+langid
+llvmlite
+Mako
+Markdown
+MarkupSafe
+matplotlib
+mlflow
+mock
+numba
+numpy
+nvidia-cublas-cu11
+nvidia-cuda-nvrtc-cu11
+nvidia-cuda-runtime-cu11
+nvidia-cudnn-cu11
+oauthlib
+packaging
+pandas
+paramiko
+Pillow
+pluggy
+protobuf==3.20.*
+pyarrow
+pycparser
+PyJWT
+PyNaCl
+pyparsing
+pysftp
+pytest
+pytest-dotenv
+pytest-env
+python-dateutil
+python-dotenv
+pytz
+PyYAML
+qanary-helpers
+querystring-parser
+rdflib
+regex
+requests
+scikit-learn
+scipy
+sentencepiece
+shap
+six
+slicer
+smmap
+SPARQLWrapper
+SQLAlchemy
+sqlparse
+tabulate
+threadpoolctl
+tokenizers
+tomli
+torch
+tqdm
+transformers
+typing_extensions
+urllib3
+websocket-client
+Werkzeug
+zipp
```

Comment:
Are you sure that such a long list of requirements is needed? I suggest to use `pipreqs` for generating it

Reply:
I will try to reduce the list of requirements